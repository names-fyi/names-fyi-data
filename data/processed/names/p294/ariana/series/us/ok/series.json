[{"y":1989,"t":7,"m":0,"f":7,"mr":null,"fr":394,"mp":0,"fp":0.0004060089321965083},{"y":1992,"t":5,"m":0,"f":5,"mr":null,"fr":545,"mp":0,"fp":0.0002907822041291073},{"y":1994,"t":5,"m":0,"f":5,"mr":null,"fr":554,"mp":0,"fp":0.000308889849879533},{"y":1995,"t":17,"m":0,"f":17,"mr":null,"fr":204,"mp":0,"fp":0.0010844603215105894},{"y":1996,"t":11,"m":0,"f":11,"mr":null,"fr":292,"mp":0,"fp":0.0006886621173229825},{"y":1997,"t":9,"m":0,"f":9,"mr":null,"fr":363,"mp":0,"fp":0.000535459305092813},{"y":1998,"t":19,"m":0,"f":19,"mr":null,"fr":211,"mp":0,"fp":0.001123794877861241},{"y":1999,"t":13,"m":0,"f":13,"mr":null,"fr":282,"mp":0,"fp":0.0007792830595851816},{"y":2000,"t":23,"m":0,"f":23,"mr":null,"fr":175,"mp":0,"fp":0.0013709244799427788},{"y":2001,"t":21,"m":0,"f":21,"mr":null,"fr":187,"mp":0,"fp":0.0012555303120889633},{"y":2002,"t":26,"m":0,"f":26,"mr":null,"fr":155,"mp":0,"fp":0.0015653220951234196},{"y":2003,"t":25,"m":0,"f":25,"mr":null,"fr":162,"mp":0,"fp":0.0015029457737164842},{"y":2004,"t":30,"m":0,"f":30,"mr":null,"fr":141,"mp":0,"fp":0.0017800984987835994},{"y":2005,"t":32,"m":0,"f":32,"mr":null,"fr":127,"mp":0,"fp":0.0019195009297582628},{"y":2006,"t":31,"m":0,"f":31,"mr":null,"fr":142,"mp":0,"fp":0.001771934838525293},{"y":2007,"t":33,"m":0,"f":33,"mr":null,"fr":127,"mp":0,"fp":0.0018550789813929958},{"y":2008,"t":26,"m":0,"f":26,"mr":null,"fr":156,"mp":0,"fp":0.0014926229978758827},{"y":2009,"t":39,"m":0,"f":39,"mr":null,"fr":95,"mp":0,"fp":0.002269156920928609},{"y":2010,"t":41,"m":0,"f":41,"mr":null,"fr":83,"mp":0,"fp":0.0024270407861244304},{"y":2011,"t":28,"m":0,"f":28,"mr":null,"fr":136,"mp":0,"fp":0.001712014674411495},{"y":2012,"t":35,"m":0,"f":35,"mr":null,"fr":105,"mp":0,"fp":0.002127788923338805},{"y":2013,"t":40,"m":0,"f":40,"mr":null,"fr":93,"mp":0,"fp":0.0024074631357207344},{"y":2014,"t":47,"m":0,"f":47,"mr":null,"fr":76,"mp":0,"fp":0.002793296089385475},{"y":2015,"t":41,"m":0,"f":41,"mr":null,"fr":89,"mp":0,"fp":0.0024404761904761904},{"y":2016,"t":36,"m":0,"f":36,"mr":null,"fr":107,"mp":0,"fp":0.0021492537313432835},{"y":2017,"t":47,"m":0,"f":47,"mr":null,"fr":66,"mp":0,"fp":0.0029987877241115296},{"y":2018,"t":25,"m":0,"f":25,"mr":null,"fr":150,"mp":0,"fp":0.0016034891924828426},{"y":2019,"t":28,"m":0,"f":28,"mr":null,"fr":117,"mp":0,"fp":0.001826126654927281},{"y":2020,"t":32,"m":0,"f":32,"mr":null,"fr":108,"mp":0,"fp":0.00218281036834925},{"y":2021,"t":26,"m":0,"f":26,"mr":null,"fr":135,"mp":0,"fp":0.0017676252634441498},{"y":2022,"t":25,"m":0,"f":25,"mr":null,"fr":135,"mp":0,"fp":0.0017702874946891375}]