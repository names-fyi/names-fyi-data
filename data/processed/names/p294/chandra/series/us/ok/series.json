[{"y":1967,"t":5,"m":0,"f":5,"mr":null,"fr":442,"mp":0,"fp":0.0003207595586348473},{"y":1968,"t":6,"m":0,"f":6,"mr":null,"fr":396,"mp":0,"fp":0.00037535189239912417},{"y":1969,"t":8,"m":0,"f":8,"mr":null,"fr":345,"mp":0,"fp":0.0004802209016147428},{"y":1970,"t":7,"m":0,"f":7,"mr":null,"fr":383,"mp":0,"fp":0.00039752399341245955},{"y":1971,"t":7,"m":0,"f":7,"mr":null,"fr":403,"mp":0,"fp":0.0003966679888932963},{"y":1972,"t":9,"m":0,"f":9,"mr":null,"fr":315,"mp":0,"fp":0.0005518425409283218},{"y":1973,"t":7,"m":0,"f":7,"mr":null,"fr":365,"mp":0,"fp":0.00045510695013328134},{"y":1974,"t":12,"m":0,"f":12,"mr":null,"fr":266,"mp":0,"fp":0.0007574323044877864},{"y":1975,"t":21,"m":0,"f":21,"mr":null,"fr":166,"mp":0,"fp":0.001317027281279398},{"y":1976,"t":12,"m":0,"f":12,"mr":null,"fr":256,"mp":0,"fp":0.0007480364044383493},{"y":1977,"t":16,"m":0,"f":16,"mr":null,"fr":211,"mp":0,"fp":0.0009459619250325174},{"y":1978,"t":20,"m":0,"f":20,"mr":null,"fr":184,"mp":0,"fp":0.0011761246692149367},{"y":1979,"t":10,"m":0,"f":10,"mr":null,"fr":309,"mp":0,"fp":0.0005400442836312577},{"y":1980,"t":15,"m":0,"f":15,"mr":null,"fr":238,"mp":0,"fp":0.0007772826199606177},{"y":1981,"t":11,"m":0,"f":11,"mr":null,"fr":308,"mp":0,"fp":0.0005472636815920398},{"y":1982,"t":12,"m":0,"f":12,"mr":null,"fr":306,"mp":0,"fp":0.0005316556643480573},{"y":1983,"t":13,"m":0,"f":13,"mr":null,"fr":283,"mp":0,"fp":0.0005958109904211925},{"y":1984,"t":11,"m":0,"f":11,"mr":null,"fr":309,"mp":0,"fp":0.0005321464854143486},{"y":1985,"t":13,"m":0,"f":13,"mr":null,"fr":264,"mp":0,"fp":0.000646059039856873},{"y":1986,"t":12,"m":0,"f":12,"mr":null,"fr":289,"mp":0,"fp":0.0006390116619628308},{"y":1987,"t":10,"m":0,"f":10,"mr":null,"fr":318,"mp":0,"fp":0.0005580045756375203},{"y":1988,"t":6,"m":0,"f":6,"mr":null,"fr":451,"mp":0,"fp":0.00034478795540742444},{"y":1989,"t":6,"m":0,"f":6,"mr":null,"fr":452,"mp":0,"fp":0.0003480076561684357},{"y":1990,"t":7,"m":0,"f":7,"mr":null,"fr":409,"mp":0,"fp":0.0004011461318051576},{"y":1992,"t":8,"m":0,"f":8,"mr":null,"fr":385,"mp":0,"fp":0.0004652515266065717},{"y":1994,"t":5,"m":0,"f":5,"mr":null,"fr":565,"mp":0,"fp":0.000308889849879533}]