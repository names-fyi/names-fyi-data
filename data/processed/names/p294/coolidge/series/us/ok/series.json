[{"y":1925,"t":5,"m":5,"f":0,"mr":414,"fr":null,"mp":0.00021795039449021403,"fp":0}]