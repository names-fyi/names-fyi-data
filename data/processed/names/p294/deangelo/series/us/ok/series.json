[{"y":1973,"t":6,"m":6,"f":0,"mr":343,"fr":null,"mp":0.00033046926635822867,"fp":0},{"y":1975,"t":5,"m":5,"f":0,"mr":380,"fr":null,"mp":0.0002679097679901409,"fp":0},{"y":1976,"t":7,"m":7,"f":0,"mr":315,"fr":null,"mp":0.00036624287134411136,"fp":0},{"y":1978,"t":6,"m":6,"f":0,"mr":359,"fr":null,"mp":0.00030217566478646255,"fp":0},{"y":1979,"t":8,"m":8,"f":0,"mr":307,"fr":null,"mp":0.0003747072599531616,"fp":0},{"y":1980,"t":5,"m":5,"f":0,"mr":411,"fr":null,"mp":0.00021781746896101068,"fp":0},{"y":1983,"t":6,"m":6,"f":0,"mr":389,"fr":null,"mp":0.0002395879087968694,"fp":0},{"y":1984,"t":8,"m":8,"f":0,"mr":325,"fr":null,"mp":0.0003322259136212625,"fp":0},{"y":1985,"t":9,"m":9,"f":0,"mr":291,"fr":null,"mp":0.00038150142003306347,"fp":0},{"y":1986,"t":11,"m":11,"f":0,"mr":250,"fr":null,"mp":0.0004893020773097282,"fp":0},{"y":1988,"t":12,"m":12,"f":0,"mr":240,"fr":null,"mp":0.0005787316132143718,"fp":0},{"y":1989,"t":5,"m":5,"f":0,"mr":451,"fr":null,"mp":0.00024330900243309004,"fp":0},{"y":1990,"t":7,"m":7,"f":0,"mr":357,"fr":null,"mp":0.00034098105119586923,"fp":0},{"y":1991,"t":10,"m":10,"f":0,"mr":279,"fr":null,"mp":0.0004836525440123815,"fp":0},{"y":1992,"t":6,"m":6,"f":0,"mr":418,"fr":null,"mp":0.00029510131811922094,"fp":0},{"y":1993,"t":6,"m":6,"f":0,"mr":403,"fr":null,"mp":0.00030762920426579164,"fp":0},{"y":1995,"t":11,"m":11,"f":0,"mr":281,"fr":null,"mp":0.0005659309564233164,"fp":0},{"y":1996,"t":6,"m":6,"f":0,"mr":433,"fr":null,"mp":0.00031481189988981584,"fp":0},{"y":1997,"t":5,"m":5,"f":0,"mr":522,"fr":null,"mp":0.00024865725084543465,"fp":0},{"y":1999,"t":9,"m":9,"f":0,"mr":371,"fr":null,"mp":0.00044494981954812873,"fp":0},{"y":2000,"t":6,"m":6,"f":0,"mr":479,"fr":null,"mp":0.0002911208151382824,"fp":0},{"y":2004,"t":7,"m":7,"f":0,"mr":473,"fr":null,"mp":0.0003385895327464448,"fp":0},{"y":2005,"t":6,"m":6,"f":0,"mr":537,"fr":null,"mp":0.0002910360884749709,"fp":0},{"y":2011,"t":6,"m":6,"f":0,"mr":603,"fr":null,"mp":0.00029523200314914136,"fp":0},{"y":2014,"t":5,"m":5,"f":0,"mr":726,"fr":null,"mp":0.00024256537136758356,"fp":0}]