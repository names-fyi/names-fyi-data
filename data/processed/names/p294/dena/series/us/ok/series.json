[{"y":1918,"t":6,"m":0,"f":6,"mr":null,"fr":459,"mp":0,"fp":0.00025708042332576374},{"y":1919,"t":5,"m":0,"f":5,"mr":null,"fr":489,"mp":0,"fp":0.00022517451024544022},{"y":1921,"t":5,"m":0,"f":5,"mr":null,"fr":529,"mp":0,"fp":0.000190056256651969},{"y":1923,"t":5,"m":0,"f":5,"mr":null,"fr":531,"mp":0,"fp":0.00019727756954034326},{"y":1925,"t":6,"m":0,"f":6,"mr":null,"fr":470,"mp":0,"fp":0.00023200958972970884},{"y":1927,"t":5,"m":0,"f":5,"mr":null,"fr":492,"mp":0,"fp":0.00020118295577998632},{"y":1928,"t":5,"m":0,"f":5,"mr":null,"fr":482,"mp":0,"fp":0.00021185542985466718},{"y":1929,"t":5,"m":0,"f":5,"mr":null,"fr":470,"mp":0,"fp":0.0002192790106131041},{"y":1930,"t":5,"m":0,"f":5,"mr":null,"fr":479,"mp":0,"fp":0.0002104111433741531},{"y":1931,"t":6,"m":0,"f":6,"mr":null,"fr":430,"mp":0,"fp":0.00027326137450471376},{"y":1932,"t":6,"m":0,"f":6,"mr":null,"fr":422,"mp":0,"fp":0.000273211602386048},{"y":1939,"t":10,"m":0,"f":10,"mr":null,"fr":296,"mp":0,"fp":0.0004995504046358277},{"y":1943,"t":6,"m":0,"f":6,"mr":null,"fr":394,"mp":0,"fp":0.0002810304449648712},{"y":1946,"t":7,"m":0,"f":7,"mr":null,"fr":359,"mp":0,"fp":0.00032216494845360824},{"y":1947,"t":6,"m":0,"f":6,"mr":null,"fr":409,"mp":0,"fp":0.00026006675046595293},{"y":1948,"t":6,"m":0,"f":6,"mr":null,"fr":396,"mp":0,"fp":0.00027739251040221914},{"y":1949,"t":7,"m":0,"f":7,"mr":null,"fr":361,"mp":0,"fp":0.00032559653937392437},{"y":1950,"t":8,"m":0,"f":8,"mr":null,"fr":332,"mp":0,"fp":0.00036953208000369534},{"y":1951,"t":6,"m":0,"f":6,"mr":null,"fr":396,"mp":0,"fp":0.0002781770133061338},{"y":1952,"t":8,"m":0,"f":8,"mr":null,"fr":333,"mp":0,"fp":0.0003724568182876298},{"y":1953,"t":13,"m":0,"f":13,"mr":null,"fr":237,"mp":0,"fp":0.0005898634239303054},{"y":1954,"t":10,"m":0,"f":10,"mr":null,"fr":304,"mp":0,"fp":0.0004436360409919702},{"y":1955,"t":14,"m":0,"f":14,"mr":null,"fr":233,"mp":0,"fp":0.0006347766946270687},{"y":1956,"t":21,"m":0,"f":21,"mr":null,"fr":198,"mp":0,"fp":0.0009419574773481654},{"y":1957,"t":15,"m":0,"f":15,"mr":null,"fr":236,"mp":0,"fp":0.0006864360241625481},{"y":1958,"t":21,"m":0,"f":21,"mr":null,"fr":192,"mp":0,"fp":0.0009849906191369607},{"y":1959,"t":11,"m":0,"f":11,"mr":null,"fr":284,"mp":0,"fp":0.0005157055789967183},{"y":1960,"t":29,"m":0,"f":29,"mr":null,"fr":161,"mp":0,"fp":0.0013829279923700524},{"y":1961,"t":30,"m":0,"f":30,"mr":null,"fr":160,"mp":0,"fp":0.0014394702749388226},{"y":1962,"t":19,"m":0,"f":19,"mr":null,"fr":216,"mp":0,"fp":0.0009217484111968175},{"y":1963,"t":22,"m":0,"f":22,"mr":null,"fr":198,"mp":0,"fp":0.0010925162635943785},{"y":1964,"t":18,"m":0,"f":18,"mr":null,"fr":217,"mp":0,"fp":0.0009517766497461929},{"y":1965,"t":25,"m":0,"f":25,"mr":null,"fr":162,"mp":0,"fp":0.0014685150375939849},{"y":1966,"t":27,"m":0,"f":27,"mr":null,"fr":136,"mp":0,"fp":0.0017273367027061607},{"y":1967,"t":27,"m":0,"f":27,"mr":null,"fr":134,"mp":0,"fp":0.0017321016166281756},{"y":1968,"t":21,"m":0,"f":21,"mr":null,"fr":171,"mp":0,"fp":0.0013137316233969346},{"y":1969,"t":39,"m":0,"f":39,"mr":null,"fr":102,"mp":0,"fp":0.002341076895371871},{"y":1970,"t":27,"m":0,"f":27,"mr":null,"fr":161,"mp":0,"fp":0.0015333068317337725},{"y":1971,"t":19,"m":0,"f":19,"mr":null,"fr":205,"mp":0,"fp":0.0010766702555675186},{"y":1972,"t":23,"m":0,"f":23,"mr":null,"fr":164,"mp":0,"fp":0.0014102642712612668},{"y":1973,"t":15,"m":0,"f":15,"mr":null,"fr":225,"mp":0,"fp":0.0009752291788570314},{"y":1974,"t":18,"m":0,"f":18,"mr":null,"fr":186,"mp":0,"fp":0.0011361484567316796},{"y":1975,"t":10,"m":0,"f":10,"mr":null,"fr":292,"mp":0,"fp":0.0006271558482282847},{"y":1976,"t":14,"m":0,"f":14,"mr":null,"fr":234,"mp":0,"fp":0.0008727091385114076},{"y":1977,"t":9,"m":0,"f":9,"mr":null,"fr":323,"mp":0,"fp":0.000532103582830791},{"y":1978,"t":20,"m":0,"f":20,"mr":null,"fr":186,"mp":0,"fp":0.0011761246692149367},{"y":1979,"t":9,"m":0,"f":9,"mr":null,"fr":338,"mp":0,"fp":0.000486039855268132},{"y":1980,"t":14,"m":0,"f":14,"mr":null,"fr":254,"mp":0,"fp":0.0007254637786299099},{"y":1981,"t":14,"m":0,"f":14,"mr":null,"fr":256,"mp":0,"fp":0.0006965174129353234},{"y":1982,"t":10,"m":0,"f":10,"mr":null,"fr":366,"mp":0,"fp":0.0004430463869567144},{"y":1983,"t":6,"m":0,"f":6,"mr":null,"fr":508,"mp":0,"fp":0.00027498968788670426},{"y":1984,"t":5,"m":0,"f":5,"mr":null,"fr":570,"mp":0,"fp":0.0002418847660974312},{"y":1985,"t":9,"m":0,"f":9,"mr":null,"fr":359,"mp":0,"fp":0.0004472716429778352},{"y":1986,"t":7,"m":0,"f":7,"mr":null,"fr":414,"mp":0,"fp":0.0003727568028116513},{"y":1988,"t":5,"m":0,"f":5,"mr":null,"fr":533,"mp":0,"fp":0.0002873232961728537},{"y":1991,"t":5,"m":0,"f":5,"mr":null,"fr":522,"mp":0,"fp":0.0002931691586045148},{"y":1994,"t":5,"m":0,"f":5,"mr":null,"fr":576,"mp":0,"fp":0.000308889849879533}]