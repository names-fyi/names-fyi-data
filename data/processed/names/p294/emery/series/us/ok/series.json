[{"y":1910,"t":6,"m":6,"f":0,"mr":129,"fr":null,"mp":0.0018552875695732839,"fp":0},{"y":1912,"t":8,"m":8,"f":0,"mr":176,"fr":null,"mp":0.0011185682326621924,"fp":0},{"y":1913,"t":5,"m":5,"f":0,"mr":276,"fr":null,"mp":0.0005675368898978433,"fp":0},{"y":1914,"t":6,"m":6,"f":0,"mr":266,"fr":null,"mp":0.0005608000747733433,"fp":0},{"y":1915,"t":5,"m":5,"f":0,"mr":375,"fr":null,"mp":0.0003297500494625074,"fp":0},{"y":1916,"t":11,"m":11,"f":0,"mr":228,"fr":null,"mp":0.0007072135785007072,"fp":0},{"y":1917,"t":5,"m":5,"f":0,"mr":378,"fr":null,"mp":0.0003007518796992481,"fp":0},{"y":1918,"t":9,"m":9,"f":0,"mr":298,"fr":null,"mp":0.00045817848597464746,"fp":0},{"y":1919,"t":8,"m":8,"f":0,"mr":305,"fr":null,"mp":0.0004260077746418872,"fp":0},{"y":1920,"t":10,"m":10,"f":0,"mr":288,"fr":null,"mp":0.0004601297565913588,"fp":0},{"y":1921,"t":5,"m":5,"f":0,"mr":443,"fr":null,"mp":0.00021760891326108716,"fp":0},{"y":1922,"t":9,"m":9,"f":0,"mr":315,"fr":null,"mp":0.00040709245521982994,"fp":0},{"y":1923,"t":8,"m":8,"f":0,"mr":318,"fr":null,"mp":0.00035656979853806385,"fp":0},{"y":1925,"t":7,"m":7,"f":0,"mr":345,"fr":null,"mp":0.0003051305522862996,"fp":0},{"y":1926,"t":7,"m":7,"f":0,"mr":351,"fr":null,"mp":0.00030895528975592533,"fp":0},{"y":1927,"t":6,"m":6,"f":0,"mr":370,"fr":null,"mp":0.0002611875326484416,"fp":0},{"y":1928,"t":10,"m":10,"f":0,"mr":271,"fr":null,"mp":0.00043738791934566765,"fp":0},{"y":1929,"t":9,"m":9,"f":0,"mr":280,"fr":null,"mp":0.00041240892636209506,"fp":0},{"y":1930,"t":5,"m":5,"f":0,"mr":391,"fr":null,"mp":0.0002148781640809661,"fp":0},{"y":1932,"t":6,"m":6,"f":0,"mr":349,"fr":null,"mp":0.00027620494406849883,"fp":0},{"y":1933,"t":9,"m":9,"f":0,"mr":281,"fr":null,"mp":0.00041152263374485596,"fp":0},{"y":1935,"t":7,"m":7,"f":0,"mr":328,"fr":null,"mp":0.0003213810201551811,"fp":0},{"y":1942,"t":8,"m":8,"f":0,"mr":281,"fr":null,"mp":0.00037250884708511827,"fp":0},{"y":1944,"t":5,"m":5,"f":0,"mr":356,"fr":null,"mp":0.00023195398033030247,"fp":0},{"y":1952,"t":5,"m":5,"f":0,"mr":372,"fr":null,"mp":0.0002078051618802211,"fp":0},{"y":2004,"t":10,"m":0,"f":10,"mr":null,"fr":386,"mp":0,"fp":0.0005933661662611998},{"y":2005,"t":9,"m":0,"f":9,"mr":null,"fr":448,"mp":0,"fp":0.0005398596364945115},{"y":2006,"t":7,"m":0,"f":7,"mr":null,"fr":560,"mp":0,"fp":0.00040011431837667905},{"y":2007,"t":22,"m":5,"f":17,"mr":658,"fr":258,"mp":0.00022966331358228838,"fp":0.0009556467479903311},{"y":2008,"t":25,"m":7,"f":18,"mr":569,"fr":242,"mp":0.00032688895115345104,"fp":0.0010333543831448418},{"y":2009,"t":24,"m":5,"f":19,"mr":710,"fr":238,"mp":0.00023326335432703523,"fp":0.0011054867050677838},{"y":2010,"t":33,"m":0,"f":33,"mr":null,"fr":121,"mp":0,"fp":0.0019534718522464926},{"y":2011,"t":43,"m":6,"f":37,"mr":608,"fr":99,"mp":0.00029523200314914136,"fp":0.0022623051054723327},{"y":2012,"t":44,"m":5,"f":39,"mr":703,"fr":92,"mp":0.0002471088267272907,"fp":0.0023709648002918113},{"y":2013,"t":50,"m":0,"f":50,"mr":null,"fr":69,"mp":0,"fp":0.003009328919650918},{"y":2014,"t":41,"m":0,"f":41,"mr":null,"fr":92,"mp":0,"fp":0.002436705099251159},{"y":2015,"t":72,"m":0,"f":72,"mr":null,"fr":33,"mp":0,"fp":0.004285714285714286},{"y":2016,"t":68,"m":0,"f":68,"mr":null,"fr":41,"mp":0,"fp":0.004059701492537314},{"y":2017,"t":54,"m":6,"f":48,"mr":634,"fr":64,"mp":0.00031104199066874026,"fp":0.003062591718241562},{"y":2018,"t":70,"m":5,"f":65,"mr":705,"fr":39,"mp":0.0002667235676944415,"fp":0.004169071900455391},{"y":2019,"t":52,"m":0,"f":52,"mr":null,"fr":54,"mp":0,"fp":0.003391378073436379},{"y":2020,"t":49,"m":0,"f":49,"mr":null,"fr":52,"mp":0,"fp":0.0033424283765347887},{"y":2021,"t":55,"m":5,"f":50,"mr":733,"fr":52,"mp":0.0002819601872215643,"fp":0.0033992793527772114},{"y":2022,"t":38,"m":0,"f":38,"mr":null,"fr":82,"mp":0,"fp":0.002690836991927489}]