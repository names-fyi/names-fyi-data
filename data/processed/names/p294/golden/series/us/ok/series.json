[{"y":1921,"t":5,"m":0,"f":5,"mr":null,"fr":538,"mp":0,"fp":0.000190056256651969}]